README.md

```diff
@@ -1,32 +1,37 @@
 # Pants Build System
 
-Pants is a build system for software projects in a variety of languages.
-It works particularly well for a source code repository that contains
-many distinct projects.
+Pants is a scalable build system for _monorepos_: codebases containing multiple projects, often using multiple programming languages and frameworks, in a single unified code repos
```

Comment:
This sentence ends strangely. Should maybe be in `"in a single repository"` or something.